test/acceptance/button-subtle.gspec

```diff
@@ -0,0 +1,153 @@
+@objects
```

Comment:
If you run the `npm run dump` command, Galen should add some new screen captures to the `test/acceptance/dumps` directory. Those should get into source control, so we can re-run it later and see if there are any unexpected diffs.